web-app/src/app/routes/alert/alert-notice/alert-notice.component.ts

```diff
@@ -348,6 +353,17 @@ export class AlertNoticeComponent implements OnInit {
   }
 
   onManageReceiverModalOk() {
+    // @ts-ignore
+    if (this.receiverForm.invalid) {
+      // @ts-ignore
+      Object.values(this.receiverForm.controls).forEach(control => {
```

Comment:
hi how about here `@ts-ignore` and below

Reply:
forgot to change some parts, I have changed them